crates/oxc_linter/src/snapshots/eslint_prefer_numeric_literals.snap

```diff
@@ -88,7 +88,7 @@ source: crates/oxc_linter/src/tester.rs
   ⚠ eslint(prefer-numeric-literals): Use hexadecimal literals instead of parseInt().
    ╭─[prefer_numeric_literals.tsx:1:1]
  1 │ Number.parseInt('1️⃣3️⃣3️⃣7️⃣', 16);
-   · ───────────────────────────
+   · ───────────────────────────────
```

Comment:
:sigh: i thought i fixed this, but some of these are still off by one 🫠

Reply:
both of those look correct? or are they not?

Reply:
The first one does not include the `]`

Reply:
ah github just isn't showing it to me